centralizedmetrics/providers/appsflyer.go

```diff
@@ -0,0 +1,118 @@
+package providers
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/ethereum/go-ethereum/log"
+
+	"github.com/status-im/status-go/centralizedmetrics/common"
+)
+
+const AppsflyerBaseURL = "https://api3.appsflyer.com"
+
+var AppsflyerAppID = ""
+var AppsflyerToken = ""
+
+// AppsflyerMetricProcessor implements MetricProcessor for Appsflyer
+type AppsflyerMetricProcessor struct {
+	AppID   string
+	Secret  string
+	BaseURL string
```

Comment:
I guess they all should be private? We set the m with constructor and never change.
Same for `MixpanelMetricProcessor`.